brave/brave-core
# Brave 24166

Show IPFS interstitial after url switching
Resolves https://github.com/brave/brave-browser/issues/24188
At the moment IPFS interstitial for the "Ask" option is shown
only when navigation is started and url has ipfs scheme.
If url changes after that(for ex. after UD or ENS resoulution)
interstitial won't be shown after that.

Add error code for ipfs navigations when ipfs is disabled
Resolves https://github.com/brave/brave-browser/issues/24166


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1) Set ipfs resolve method to "Ask" and open "vitalik.eth". Check that chooser is shown.
2) Disable ipfs and open ipfs page. Check that descriptive error page is opened.
